chia/server/server.py

```diff
@@ -565,10 +563,15 @@ async def wrapped_coroutine() -> Optional[Message]:
                         f"Time taken to process {message_type} from {connection.peer_node_id} is "
                         f"{time.time() - start_time} seconds"
                     )
-
                     if response is not None:
                         response_message = Message(response.type, full_message.id, response.data)
                         await connection.reply_to_request(response_message)
+                    elif hasattr(f, MSG_REPLY_TYPE):
+                        capabilities = connection.capabilities
```

Comment:
can we initialize capabilities to an empty Dict? Seems cleaner than using None

Reply:
its a list of tuples, now initiated to en empty list